src/pad/src/RDLRouter.h

```diff
@@ -246,6 +246,9 @@ class RDLRouter
   // Debugging
   RDLGui* gui_;
   odb::dbNet* debug_net_{nullptr};
+
+  // Consts
+  static constexpr const char* kRouteProperty_ = "RDL_ROUTE";
```

Comment:
No need for the trailing _ (https://google.github.io/styleguide/cppguide.html#Constant_Names)